executor-plugins/resource-tagger/internal/service/executor_wrapper.go

```diff
@@ -21,15 +21,11 @@ import (
 	"flag"
```

Comment:
Here we should do some further refactor:
1. Rename executor-plugins/resource-tagger to default-plugins, I think we can just keep this one for now.
2. move executor_wrapper.go to pkg folder and we should keep it as stable as possible
3. move executor.go to pkg/plugins/resource-updater folder and other user can contribute their plugin to pkg/plugins/<xxxx> folder for more features.